### PR TITLE
Minor fixes to new tutorial strings

### PR DESCRIPTION
**Pull request by Faalagorn**
#### Summary
None

#### Purpose of change
Fix typos and change hardcoded 'c' string with actual keybinding introduced by #55230

#### Describe the solution
Fixing typos, including an omission of one string that is in old file.

#### Describe alternatives you've considered
None

#### Testing
Just the name changes, but to test it start the game, start tutorial and go to the room where message appears, preferably with keybindings changed

#### Additional context
Found when updating translations in Transifex, this will fix an issue I left there.

### CODE REVIEW

**Comment by Faalagorn** [comment]
There are some strings that aren't using cyan color, e.g. _There is a display rack next to you with a **steel spear** lying on it._ or _To continue the tutorial, please wear the **spear strap**. or _Press <keybind:apply> to open the <color_light_cyan>Use item</color> menu, then select the **spear strap**, then select the **spear**,_ possibly more but not sure if it's a good idea to make them all cyan.

I see that in #55230 @haveric changed some instances of "spear" to "steel speare", what was the motive behind these specific strings and is it good idea to change the remaining ones too?

File: data/json/snippets/tutorial.json [hunk 1]
```diff
@@ -185,7 +185,7 @@
...
+        "text": "Looks like you're hungry!  The hunger status is indicated on the sidebar.  Let's make a <color_light_cyan>peanut butter sandwich</color> from <color_light_cyan>peanut butter</color> and <color_light_cyan>crackers</color> on the counter next to the wall.\n\nTo make food, you need to use the <color_light_cyan>Crafting</color> menu.  Press <keybind:craft> to open the menu.  Press <keybind:NEXT_TAB> and <keybind:PREV_TAB> to toggle crafting categories and select the <color_light_cyan>FOOD</color> category.  From there, select peanut butter sandwich and press <keybind:CONFIRM> to start crafting.\n\nNote that you don't need to stand directly near the counter to be able to craft.  The <color_light_cyan>crafting radius</color> is 6 tiles, so while you're standing no farther than 6 tiles from any crafting components and tools, your character is able to still get them in order to use them.\n\nTo continue the tutorial, please craft the peanut butter sandwich and then eat it by pressing <keybind:eat>."
```
Comment (Faalagorn) [inline data/json/snippets/tutorial.json:188]: Actually, it has no name in-game (the keybinding for it is called "Craft Items") so my bad, but I still think it's better that way.

**Comment by haveric** [comment]
> I see that in #55230 @haveric changed some instances of "spear" to "steel speare", what was the motive behind these specific strings and is it good idea to change the remaining ones too?

I was just trying to make it look nicer and make it more consistent. Maybe they don't all need to be the full names, but also as a tutorial, consistency is good.

File: data/json/snippets/tutorial.json [hunk 2]
```diff
@@ -157,7 +157,7 @@
...
+        "text": "Bashing things is hard work that will deplete your stamina.  You can see how much stamina you have on the sidebar.  The less stamina you have, the slower your movement and most actions will be.  To restore stamina, you need only to wait.  You can press <keybind:wait> to open the <color_light_cyan>Wait for how long</color> menu and select the amount of time you want to wait.  Select option <color_light_cyan>Wait until you catch your breath</color> to fully restore all lost stamina.\n\nBe careful about doing too many strenuous activities as you will build up your weariness over time, which will stack movement and action penalties on top of your stamina cost.  You can watch your <color_light_cyan>Weary Transition</color> in the sidebar as you are performing longer activities to see their effect on  your weariness.  To reduce your weariness, you just need to rest or do less strenuous activities for a while."
```
Comment (haveric) [inline data/json/snippets/tutorial.json:160]: I could have sworn it said Weariness, but maybe my brain autofilled it in to sound nicer. Good catch.

File: data/json/snippets/tutorial.json [hunk 3]
```diff
@@ -145,7 +145,7 @@
...
+        "text": "From a water source like this, you can fill any containers you might have.  Examine the dispenser by pressing <keybind:examine>.  Next, press <keybind:close>) to fill a container, then select the <color_light_cyan>plastic bottle</color> to put the water into.\n\nThe Examine command (<keybind:examine> can be used for interacting with many types of terrain and furniture.\n\nTo continue the tutorial, please fill the plastic bottle with water, drink a little of it (by pressing <keybind:eat> and selecting <color_light_cyan>clean water</color>), and then move to the corridor to the north."
```
[the remark this answers is not in the document]
Comment (haveric) [reply data/json/snippets/tutorial.json]: It does look like there's a dropped ( here though

**Comment by Faalagorn** [comment]
> > I see that in #55230 @haveric changed some instances of "spear" to "steel speare", what was the motive behind these specific strings and is it good idea to change the remaining ones too?
> 
> I was just trying to make it look nicer and make it more consistent. Maybe they don't all need to be the full names, but also as a tutorial, consistency is good.

I think so too, but do you think I should add it to every other instance as well, or will it be too excessive?

**Comment by haveric** [comment]
> I think so too, but do you think I should add it to every other instance as well, or will it be too excessive?

Totally up to you, but I'd suggest just trying it and if it ends up looking too polluted of colored strings, maybe some of the sentences could be tweaked/reworded to reduce them.

**Comment by Faalagorn** [comment]
I decided I'll keep the highlighting for later, as I don't want to drag this simple PR more than needed, as it's already became messy. 
Sadly I didn't have time to test it proper but should be fine as it's just text being changed after removing the binding, but feel free to check and squash PR as I think I can't do that from GitHub web info, but hit me if you need me to do something!

**Pull request merged**